benchmark/test/test_framework.py.in

```diff
@@ -58,7 +71,7 @@ def sanitize_json(parsed_input, sanitize_all: bool = False):
 
     if isinstance(parsed_input, dict):
         return {
-            key: sanitize_json_key_value(key, value, sanitize_all)
+            sanitize_json_key(key): sanitize_json_key_value(key, value, sanitize_all)
```

Comment:
maybe rename `sanitize_json_key_value` to `sanitize_json_value`

Reply:
right, the fact that the key is involved is already present in the parameters.